### ISSUE TITLE
Main menu accessibility, lacks of WCAG 2.1 success criteria 2.4.3 focus order

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#21005]
## Context

Main menu is on the left side of the page, you can navigate Matomo routes, they are divided intro macro-categories using your mouse collapses macro categories and submenus expose routes.

## Expected Behavior
Beside "Skip to content" link should be a "Skip to menu" link too.
Then, when you select by keyboard: skip to content, your focus must brought to `<h1>` starting tag. 
But, when you select "skip to menu", the focus must go to the first macro-item of the menu, the first category, forward (tab) and backward  (SHIFT+TAB) navigation should iterate macrocategories, then enter one of them, pressing enter.
When you enter the category you must be able to iterate their routes.

Finally you must be able to exit the second level menu and return to categories. If can't be done, navigation should be linear from first item to last.

This is an example of iterable menu.

```html
<dl>
  <dt>Dashboard</dt>
  <dd>
    <ul>
     <li>My Dashboard 1</li>
     <li>My Dashboard 2</li>
   </ul>
  </dd>
  <dt>Visitors</dt>
  <dd>
    <ul>
     <li>Overview</li>
     <li>Visitor logs</li>
    <li>....</li>
   </ul>
  </dd>
</dl>
```

## Current Behavior

Actually, when you "skip to content" you must do SHIFT+TAB (select previous) to enter the final macro category of the menu.  
When you continue selecting backward, you can only choose macro categories. Hitting enter, expands the macro category but don't enter the list of sub-items.  
Result, you can't never hit any route. 

## Steps to Reproduce (for Bugs)

1. open Demo page
2. press tab to get "skip content"
3. press "shift tab" to go back in the menu
4. try menu's keyboard navigation

https://www.w3.org/WAI/WCAG21/quickref/#focus-order

## Your Environment
* Matomo Version: 4.x
* PHP Version: 8.1
* Server Operating System: any
* Browser: any
* Operating System: any
